src/jsonapi-vuex.js

```diff
@@ -359,6 +359,37 @@ const jsonapiModule = (api, conf = {}) => {
 
 // Helper methods
 
+const addJvHelpers = (obj) => {
+  // Add Utility functions to _jv child object
+  Object.assign(obj[jvtag], {
+    is_rel(name) {
+      return name in obj[jvtag]['relationships']
```

Comment:
Is this testing presence in an object? I think the recommended way is `[hasOwnProperty](https://stackoverflow.com/a/455366)`.